### PR TITLE
Prac 08 pull request feedback

### PR DESCRIPTION
**Pull request by DianaBukeikhanova**
@TinNilarSoe
@ChristineYamin

Please check my codes.

### CODE REVIEW

File: prac_08/squaring.kv [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            font_size: 48
+            multiline: False
+        Button:
+            text: 'Square'
+            on_press: app.handle_calculate(input_number.text)
+        Label:
+            id: output_label
+            font_size: 48
+            color: (1, 0.7, 0.7, 1)
+    BoxLayout:
```
Comment (ChristineYamin) [inline prac_08/squaring.kv]: I think that no need to write BoxLayout again

File: prac_08/dynamic_labels.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    def create_labels(self):
+        """Create buttons from data and add them to the GUI."""
+        for name in self.names:
+            label = Label(text=name)
+            self.root.ids.main.add_widget(label)
+
+
+if __name__ == '__main__':
+    DynamicLabelsApp().run()
```
Comment (ChristineYamin) [inline prac_08/dynamic_labels.py]: Nice coding!

File: prac_08/squaring.kv (hunk 1 above)
Comment (DianaBukeikhanova) [reply prac_08/squaring.kv]: Noted! Thank you

File: prac_08/squaring.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    def handle_calculate(self, value):
+        """Handle ValueError, output squared number."""
+        try:
+            result = float(value) ** 2
+            self.root.ids.output_label.text = str(result)
+        except ValueError:
+            pass
+
+
+SquareNumberApp().run()
```
Comment (TinNilarSoe) [inline prac_08/squaring.py:25]: Great code.

File: prac_08/convert_miles_km.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+
+class ConvertMilesApp(App):
+    """App for converting miles to kilometres."""
+    output_text = StringProperty()
+
+    def build(self):
+        """Use Kivy file to create Kivy app."""
+        self.title = "Convert miles to kilometres"
+        self.root = Builder.load_file('convert_miles_km.kv')
```
Comment (TinNilarSoe) [inline prac_08/convert_miles_km.py:16]: I think you need to add "self.output_text = '32.1868'" to show that you have used StringProperty().

Comment (ChristineYamin) [reply prac_08/convert_miles_km.py:16]: Noted !
Thanks you

File: prac_08/dynamic_labels.py (hunk 2 above)
Comment (TinNilarSoe) [inline prac_08/dynamic_labels.py]: Good code!

**Pull request merged**